system-status.sh

```diff
@@ -39,6 +39,9 @@ log minikube_ssh journalctl --boot
 log df -hT
 log minikube_ssh df -hT
 
+# get the rbd-nbd logs from the VM and write them to stdout
+log minikube_ssh find /var/log/ceph/ -type f | xargs tail -n +1
```

Comment:
why not use the same command as what is used below for rook log?

Reply:
also, this won't work as expected. `minikube_ssh` will run `find /var/log/ceph/ -type f` inside the VM, but `xargs tail -n +1` is running on the host where the files are not available

Reply:
Ans1:  See the difference:

```
[pkalever@localhost logs]$ sudo tar c /tmp/logs/ | tar xvO
tar: Removing leading `/' from member names
tmp/logs/
tmp/logs/log2
I might have failed!
tmp/logs/log1
I might have succeeded!
[pkalever@localhost logs]$ find /tmp/logs/ -type f | xargs tail -n +1
==> /tmp/logs/log2 <==
I might have failed!

==> /tmp/logs/log1 <==
I might have succeeded!
[pkalever@localhost logs]$ 
```


Ans2: I will club them by with quotations \" find /var/log/ceph/ -type f | xargs tail -n +1 \"


thanks @nixpanic